src/components/dictionaryConcepts/containers/DictionaryMappings.jsx

```diff
@@ -0,0 +1,153 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import autoBind from 'react-autobind';
+import matchSorter from 'match-sorter';
+import { Link } from 'react-router-dom';
+import ReactTable from 'react-table';
+import { connect } from 'react-redux';
+import SideNav from '../components/Sidenav';
+import Loader from '../../Loader';
+import {
+  filterBySource,
+  filterByClass,
+} from '../../../redux/actions/concepts/dictionaryConcepts';
+import {
+  fetchDictionaryMappings,
+} from '../../../redux/actions/dictionaries/dictionaryActionCreators';
+import { fetchMemberStatus } from '../../../redux/actions/user/index';
+
+export class DictionaryMappings extends Component {
+  static propTypes = {
+    match: PropTypes.shape({
+      params: PropTypes.shape({
+        typeName: PropTypes.string,
+        collectionName: PropTypes.string,
+        type: PropTypes.string,
+        dictionaryName: PropTypes.string,
+      }),
+    }).isRequired,
+    location: PropTypes.shape({
+      pathname: PropTypes.string,
+    }).isRequired,
+    mappings: PropTypes.arrayOf(PropTypes.shape({
+      from_concept_name: PropTypes.string,
+      to_concept_name: PropTypes.string,
+      map_type: PropTypes.string,
+      source: PropTypes.string,
+    })).isRequired,
+    loading: PropTypes.bool.isRequired,
+    fetchDictionaryMappings: PropTypes.func.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      mappingLimit: 10,
+    };
+    autoBind(this);
```

Comment:
Do we need this `autoBind(this)` when we are using arrow functions?

Reply:
We actually don't need it @collinewait. I have removed it. Thanks for pointing it out